### DISCUSSION TITLE
Approving Github App permissions change

### DISCUSSION THREAD

**Discussion by hcwong** [community/community#23012] [New to GitHub]
<p>Hello, </p>
<p>A GH app recently made some permission changes and requested more permissions. As the repo owner, how can I approve these additional changes the app is requesting? Thanks!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/hcwong">@hcwong</a>,</p>
<p>To see and manage Apps which are already installed on the repository – visit the repository’s settings page:</p>
<p><a href="https://github.com/ORG_NAME/REPO_NAME/settings/installations" rel="nofollow noopener">https://github.com/ORG_NAME/REPO_NAME/settings/installations.</a></p>
<p>You’ll see a list of Apps which have been installed on the repository, and a Configure button for each of those.</p>
